src/features/island/Plots/Soil.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useRef } from "react";
```

Comment:
Do we need a duplicate `Soil.tsx` component or can we reuse the existing one?

I feel the functionality in this component hasn't changed yet.